Rakefile

```diff
@@ -72,7 +73,7 @@ end
 namespace :test do
   task all: [:main,
              :rails, :railsredis, :railssidekiq, :railsactivejob,
-             :elasticsearch, :http, :sidekiq, :sinatra, :monkey]
+             :elasticsearch, :http, :sidekiq, :sinatra, :monkey, :mysql2]
```

Comment:
As this is currently written right now, this won't actually run tests in our CI suite because our current setup for CI is a bit complicated.

To get this working we need to:

 - Remove this from this line, add it to the complimentary `rspec` task instead (inserted alphabetically)
 - Add the `mysql2` gem to each `appraise 'contrib-old'` or `appraise 'contrib'` section of `Appraisals` for each version of Ruby this integration is meant to support (1.9.3-2.4.4).
 - Add a corresponding `sh 'appraisal contrib-old rake spec:mysql2'` or `sh 'appraisal contrib rake spec:mysql2'` to the CI rake task in `Rakefile` for each version the integration is meant to support, inserted alphabetically.

After that is done, you should be able to run CI tests locally from your branch by running `docker-compose run --rm tracer-2.3`, then in the container, do the following to run the tests:

 1. `bundle install`
 2. `bundle exec appraisal install`
 3. `bundle exec rake ci` or the `mysql2` tests only via `bundle exec appraisal contrib rake spec:mysql2`